assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/domain/member/service/EmailService.java

```diff
@@ -0,0 +1,67 @@
+package com.splab.invitation.domain.member.service;
+
+import com.splab.invitation.domain.member.entity.Member;
+import com.splab.invitation.domain.member.service.dto.InviteMemberCommand;
+import com.splab.invitation.persistence.member.MemberRepository;
+import jakarta.mail.Message.RecipientType;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class EmailService {
+
+  private final MemberRepository repository;
+  private final JavaMailSender mailSender;
+  private final ResourceLoader resourceLoader;
+  private static final String prefix = "http://localhost:8080/members/joinMember?";
+  private static final String CONTENT_PLACEHOLDER = "CONTENT";
+  private static final String LINK_PLACEHOLDER = "YOUR_INVITATION_LINK";
+
+  public void sendEmail(InviteMemberCommand command, String passKey) {
+    // 링크 생성
+    String link = prefix + "email=" + command.getEmail() + "&passKey=" + passKey;
+
+    // 이메일 전송
+    try {
+      MimeMessage mail = createMailContent(command, link);
+      mailSender.send(mail);
+
+    } catch (Exception e) {
+      e.printStackTrace();
+      throw new RuntimeException("메일 전송 실패");
+    }
+
+  }
+
+  private MimeMessage createMailContent(InviteMemberCommand command, String link)
+      throws Exception {
+    String to = command.getEmail();
+    String content = command.getContent();
```

Comment:
불필요한 command 객체의 재사용을 막는 것이 좋아보입니다.
```suggestion
  private MimeMessage createMailContent(to String, content String, String link)
      throws Exception {
```